### ISSUE TITLE
How to deploy the model in CVAT while using external CVAT_HOST?

### ISSUE THREAD

**Issue by purvaja2001** [opencv/cvat#7491]
Hi, I am trying to deploy the model in GCP in CVAT. External IP for the VM instance is configured with 8080 port for CVAT.
I have done below steps while SSH into the VM instance:
Cloned the latest repo
cd cvat
Exported CVAT_HOST
sudo -E docker compose up -d
Now, how to proceed for automated Annotations on my CVAT_HOST?
I tried running the 
docker compose -f components/serverless/docker-compose.serverless.yml up -d --build
But in the CVAT, I am unable to the Models tab.
Can anyone help me here please!.

**Comment by bsekachev** [comment]
> I tried running the
docker compose -f components/serverless/docker-compose.serverless.yml up -d --build

Where have you seen suggestion to use this command?

Please, proceed the guide: `https://opencv.github.io/cvat/docs/manual/advanced/serverless-tutorial/`

**Comment by purvaja2001** [comment]
I have followed exact same way, Using the same Guide .
But my doubt is it my external IP is 0.0.0.0 for example. 
CVAT would be on 0.0.0.0:8080, 
after installation will the Nuclio (serverless) using that guide be on this 0.0.0.0:8070 port?? 
And will this be communicating with CVAT for getting Models tab in CVAT?
Now, for me that was the problem, I both CVAT and Nuclio are running and I deployed a model in Nuclio but Models tab in CVAT is not visible for semi automated annotations.
